src/energyplus/ForwardTranslator/ForwardTranslateOutputTableSummaryReports.cpp

```diff
@@ -0,0 +1,73 @@
+*  disclaimer in the documentation and/or other materials provided with the distribution.
+*
+*  (3) Neither the name of the copyright holder nor the names of any contributors may be used to endorse or promote products
+*  derived from this software without specific prior written permission from the respective party.
+*
+*  (4) Other than as required in clauses (1) and (2), distributions in any form of modifications or other derivative works
+*  may not use the "OpenStudio" trademark, "OS", "os", or any other confusingly similar designation without specific prior
+*  written permission from Alliance for Sustainable Energy, LLC.
+*
+*  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDER(S) AND ANY CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES,
+*  INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+*  DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER(S), ANY CONTRIBUTORS, THE UNITED STATES GOVERNMENT, OR THE UNITED
+*  STATES DEPARTMENT OF ENERGY, NOR ANY OF THEIR EMPLOYEES, BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+*  EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF
+*  USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+*  STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
+*  ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+***********************************************************************************************************************/
+
+#include "../ForwardTranslator.hpp"
+
+#include "../../model/Model.hpp"
+#include "../../model/OutputTableSummaryReports.hpp"
+#include "../../model/OutputTableSummaryReports_Impl.hpp"
+
+#include "../../utilities/idf/IdfExtensibleGroup.hpp"
+
+#include <utilities/idd/Output_Table_SummaryReports_FieldEnums.hxx>
+#include "../../utilities/idd/IddEnums.hpp"
+#include <utilities/idd/IddEnums.hxx>
+
+
+using namespace openstudio::model;
+
+using namespace std;
+
+namespace openstudio {
+
+namespace energyplus {
+
+boost::optional<IdfObject> ForwardTranslator::translateOutputTableSummaryReports( OutputTableSummaryReports & modelObject )
+{
+  IdfObject idfObject(openstudio::IddObjectType::Output_Table_SummaryReports);
+
+  m_idfObjects.push_back(idfObject);
+
+  std::vector<std::string> summaryReports = modelObject.summaryReports();
+  if (!summaryReports.empty()) {
+    for (std::string summaryReport : summaryReports) {
+      auto eg = idfObject.pushExtensibleGroup();
+      eg.setString(Output_Table_SummaryReportsExtensibleFields::ReportName, summaryReport);
+    }
+  } else {
+    auto eg = idfObject.pushExtensibleGroup();
+    eg.setString(Output_Table_SummaryReportsExtensibleFields::ReportName, "AllSummary");
```

Comment:
I see you actually handle the case where we create a default one in FT here. I'd prefer doing it in ForwardTranslator.cpp#L558 (or model API), so the user can choose to manually instantiate an `OutputTableSummaryReports` and actually disable all outputs instead of ending with "AllSummary" anyways.